actioncable/lib/action_cable/subscription_adapter/inline.rb

```diff
@@ -6,9 +6,11 @@ class Inline < Base # :nodoc:
       def initialize(*)
         super
         @subscriber_map = nil
+        @compressor = Compressor.new
```

Comment:
Since it applies to all adapters, should this be moved up to `Base#initialize`?

Reply:
I thought about it, but it might break compatibility with some existing adapter outside of Rails. If that's not a concern I can move it

Reply:
Also the threshold for all adapters is not the same, unless we set the default limit to 7.8 kilobytes, which is the maximum we can allow in PostgreSQL.

Reply:
Ah right, I was looking at the last commit but the Postgres adapter's Compressor is configured differently 👍